src/theme.js

```diff
@@ -299,7 +299,10 @@ export class AbstractTheme {
 
   getRangeOutput (input, startvalue) {
     const output = document.createElement('output')
-    output.value = startvalue || 0
+
+    setTimeout(() => {
+      output.value = startvalue || 0
+    }, 0)
```

Comment:
@germanbisurgi 
Hm, is there no other way than a setTimeout with delay 0? Can't this be bound to an event?